gormigrate.go

```diff
@@ -178,7 +194,7 @@ func (g *Gormigrate) createMigrationTableIfNotExists() error {
 		return nil
 	}
 
-	sql := fmt.Sprintf("CREATE TABLE %s (%s VARCHAR(255) PRIMARY KEY)", g.options.TableName, g.options.IDColumnName)
+	sql := fmt.Sprintf("CREATE TABLE %s (id INTEGER PRIMARY KEY AUTOINCREMENT, %s VARCHAR(255))", g.options.TableName, g.options.IDColumnName)
```

Comment:
There's still two problems with it: it's backwards incompatible and it would be hard to existing users create a PK column on this table that already has rows.

I think we should make this PR backward compatible and fix the issue, and then it will be merged on `v1`. Then, you can open a new issue or PR to discuss these improvements for `v2` (ideally, there should be migration paths).